README.md

```diff
@@ -114,17 +155,31 @@ To start or stop individual daemons, the sysvinit script can be used:
 Running unit tests
 ==================
 
-To run build and run all tests, use ctest:
+To build and run all tests, use ctest:
 
 	cd build
 	make
 	ctest -j$(nproc)
 
+(Note: Many targets built from src/test are not run using ctest.
+Targets starting with "unittest" are run in `make check` and thus can
+be run with ctest. Targets starting with "ceph_test" can not, and should
+be run by hand.)
+
+To build and run all tests and their dependencies without other
+unnecessary targets in Ceph:
+
+        cd build
+        make check -j$(nproc)
+
 To run an individual test manually, run the ctest command with -R
 (regex matching):
 
 	ctest -R [test name]
```

Comment:
If -R indicates regex matching, then the argument is not "test name" but rather "reg ex matching test name(s)".